### ISSUE TITLE
Generate single DB diagram in .png format

### ISSUE THREAD

**Issue by abitrolly** [schemaspy/schemaspy#632]
## Expected Behavior
I need a diagram of database schema stored as a .png in repository. This allows to include it in GitHub comments and docs without the need for external hosts, and also allows comparing changes graphically.

## Current Behavior
Generated diagram is in HTML and needs a separate host.

## Possible Solution
schemaspy -outpng diagram.png <connection params...>

## Context
My software archaeology with patching abandoned project often requires reversing DB schema and patching it. Having up to date picture in front of my eyes will help greatly. Comparing commits with https://pypi.org/project/imgdiff/ on DB changes will help as well.

**Comment by npetzall** [comment]
All diagrams are in either png or svg which doesn't require server. Html output is static/embedded som Java script but doesn't require any server.

**Comment by abitrolly** [comment]
How to get just the diagram picture from http://schemaspy.org/sample/relationships.html ?

**Comment by npetzall** [comment]
Something like [output]/diagrams/summary/relationship-*(png|svg) depending on `-vizjs`

**Comment by npetzall** [comment]
http://schemaspy.org/sample/diagrams/summary/relationships.real.compact.png

**Comment by abitrolly** [comment]
> http://schemaspy.org/sample/diagrams/summary/relationships.real.compact.png

Yep. This one is great. I can not find it in my `summary/` output.

**Comment by npetzall** [comment]
Do you have anything in summary, you won't get the *real* unless there are relations declared in the database. So you might have the implied instead.

**Comment by abitrolly** [comment]
`summary/` is empty.

**Comment by npetzall** [comment]
In that case there shouldnt be anything in the html report either.

**Comment by abitrolly** [comment]
This is a Ruby on Rails project. It doesn't store relationships in DB as I see from the DB dump.

There is a HTML report in `Orhphan Tables` tab with tables listed, but it is not a single .png

**Comment by npetzall** [comment]
Then you might want the rubyonrails feature enabled `-rails`.

**Comment by npetzall** [comment]
@abitrolly Was the solution `-rails` and summary/relationships.implied.(large|compact).png ?

**Comment by abitrolly** [comment]
I don't remember.
